lib/fsext/filepath_windows_test.go

```diff
@@ -0,0 +1,50 @@
+//go:build windows
```

Comment:
Unfortunately `k6 archive` can generate an archive on one OS that is run on another. So we *probably* need to add more tests about paths generated on Windows for the archive and then check if another OS can actually work with them and vice versa. 

This actually is big part of what the current test do, but it seems it might be a good idea to extend those and double check that this is enough. 

Maybe creating a full integration test where we run an archive generated from a windows machine (both per 1.20 and post if there are end result differences) and ones generated on a linux machine. Both should run on all supported OSes

Reply:
I tested this manually by creating archives on Windows with Go 1.19.6 and 1.20.1 (on this branch), and they both ran fine on Linux.

But I agree, it would be good to have integration tests that do that.

Reply:
Generally, I like the idea since it sounds reasonable. However, it should be implemented separately. WDYT?

Reply:
Anyway, the last commit contains the tests. Please let me know if I need to extend something there.